dc

```diff
@@ -13,6 +13,11 @@ for i in $(seq 1 ${#PORTAL_MODULES}); do
     COMPOSE_FILES+=" -f docker-compose.mongodb.yml -f docker-compose.accounts.yml"
   fi
 
+  # abuse module - alias "c"
```

Comment:
nit: We're running out of convenient letters, it seems. How about `u` for `abuse`?